presto-main/src/main/java/com/facebook/presto/sql/planner/iterative/MaxCardProperty.java

```diff
@@ -0,0 +1,136 @@
+/*
```

Comment:
Same - maybe in spi or a separate sats properties pacakge?

Reply:
Please see response to your previous comment.  These aren't stats by the way. They are provable logical properties.